src/swagger.yml

```diff
@@ -4282,16 +4402,73 @@ definitions:
         type: integer
         format: int32
 
+  Creator:
+    type: object
+    properties:
+      email:
+        $ref: '#/definitions/Email'
+      firstname:
+        type: string
+      lastname:
+        type: string
+
+  PublicationTemplateId:
+    type: string
+    description: The unique identifier of a template e.g. 'a4-poster', 'a7-poster-8x', 'a4-poster-preprinted', ...
+    pattern: '/^[-a-z0-9]{3,}$/'
+
+  PublicationTemplate:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/PublicationTemplateId'
+      name:
+        type: string
+
+  PublicationBriefing:
+    type: object
+    properties:
+      id:
+        $ref: '#/definitions/UUIDv4'
+      week:
+        $ref: '#/definitions/FullWeek'
+      name:
+        type: string
+        description: Nanme for briefing and publications
+      status:
+        type: string
+      success:
+        type: boolean
+      store:
+        $ref: '#/definitions/StoreCode'
+      advertised_articles:
+        $ref: '#/definitions/AdvertisedArticleSelections'
+      languages:
+        $ref: '#/definitions/Locales'
+      templates:
+        $ref: '#/definitions/PublicationTemplates'
+      reperence :
+        type: string
+        description: Optional WPT internal reference
+      created_by:
+        $ref: '#/definitions/Creator'
```

Comment:
`Creator` seems too specific. Maybe `Collaborator` or `Person`?

Reply:
Ill go for `Collaborator`